service-mesh/app-src/apps/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN mkdir /app/${APP}
 RUN touch /app/${APP}/__init__.py
 COPY demo/${APP}.py /app/${APP}/
 COPY demo/utils.py /app/${APP}/
+COPY startup.sh /app/startup.sh
 WORKDIR /app/
 
-CMD python -m ${APP}.${APP}
+CMD ./startup.sh
```

Comment:
Would it make sense to pull in all non-appmesh specific changes to the istio branch to ensure they are non-breaking?

Reply:
I think this change in particular should stay associated with this PR because it's a change for AWS. These changes were tested with the Istio example too. The fixed rebase should have removed all of the other app changes